src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, AreaCalculator {
```

Comment:
The class name 'Figure' is appropriate as it is a noun, but the interface 'AreaCalculator' should not be a verb. Consider renaming it to something like 'AreaCalculable' to follow the naming conventions for interfaces.